sal_interfaces/Scheduler/Scheduler_Events.xml

```diff
@@ -368,4 +402,576 @@
+		<Description>Observatory model: telescope configuration.</Description>
+    <item>
+        <EFDB_Name>altitudeMinpos</EFDB_Name>
+        <Description>Telescope minimum reachable altitude position.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>altitudeMaxpos</EFDB_Name>
+        <Description>Telescope maximum reachable altitude position.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>azimuthMinpos</EFDB_Name>
+        <Description>Telescope minimum reachable azimuth position.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>azimuthMaxpos</EFDB_Name>
+        <Description>Telescope maximum reachable azimuth position.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>altitudeMaxspeed</EFDB_Name>
+        <Description>Telescope maximum speed of the altitude axis.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg/s</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>altitudeAccel</EFDB_Name>
+        <Description>Telescope altitude acceleration.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg/s2</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>altitudeDecel</EFDB_Name>
+        <Description>Telescope altitude deceleration.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg/s2</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>azimuthMaxspeed</EFDB_Name>
+        <Description>Telescope maximum speed in azimuth</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg/s</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>azimuthAccel</EFDB_Name>
+        <Description>Telescope azimuth acceleration.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg/s2</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>azimuthDecel</EFDB_Name>
+        <Description>Telescope azimuth deceleration.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>deg/s2</Units>
+        <Count>1</Count>
+    </item>
+    <item>
+        <EFDB_Name>settleTime</EFDB_Name>
+        <Description>Telescope settle time.</Description>
+        <IDL_Type>double</IDL_Type>
+        <Units>second</Units>
+        <Count>1</Count>
+    </item>
+</SALEvent>
+<SALEvent>
+    <Subsystem>Scheduler</Subsystem>
+    <EFDB_Topic>Scheduler_logevent_rotatorConfig</EFDB_Topic>
+		<Description>Observatory model: camera rotator configuration.</Description>
+    <item>
+        <EFDB_Name>minpos</EFDB_Name>
```

Comment:
There is a mix of camel case and non camel case names. Please review and make sure that either all are camel case (including renaming this one to minPos) for consistency.